debug/multi-anim.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>GPS Track + Accelerations | CARTO VL</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <!-- Include CARTO VL JS -->
+    <script src="../dist/carto-vl.js"></script>
+    <!-- Include Mapbox GL JS -->
+    <script src="https://cartodb-libs.global.ssl.fastly.net/mapbox-gl/v0.44.1-carto1/mapbox-gl.js"></script>
+    <!-- Include Mapbox GL CSS -->
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.1/mapbox-gl.css" rel="stylesheet" />
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+
+        #map {
+            position: absolute;
+            height: 100%;
+            width: 100%;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="map"></div>
+    <script>
+        const map = new mapboxgl.Map({
+            container: 'map',
+            style: 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json',
+            center: [-3.589442037237177, 40.614912811223235],
+            zoom: 15,
+            dragRotate: false
+        });
+        carto.setDefaultAuth({
+            user: 'rochoa',
+            apiKey: 'default_public'
+        });
+        const s = carto.expressions;
+        const jaramaSource = new carto.source.Dataset('circuito_del_jarama');
```

Comment:
This dataset is very nice for examples, maybe we should ask @rochoa permission to put it in cartovl 😛

Reply:
Pinging @rochoa :smile: